FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -2355,7 +2365,8 @@ void OpDispatchBuilder::RCLOp(OpcodeArgs) {
     SetRFLAG<FEXCore::X86State::RFLAG_OF_RAW_LOC>(NewOF, Size - 1, true);
 
     StoreResult(GPRClass, Op, Res, -1);
-  });
+    },
+    GetSrcSize(Op) == OpSize::i32Bit ? std::make_optional(&OpDispatchBuilder::ZeroShiftResult) : std::nullopt);
```

Comment:
ZeroShiftResult checks the src size anyway, so I think we can drop the ternary no?

Reply:
The issue comes down to the non-const shift case generating an empty block for !32-bit cases in that case, so it needs to be passed nothing to avoid that.

Reply:
Right ok